docs/DeveloperGuide.md

```diff
@@ -270,14 +270,41 @@ _{Explain here how the data archiving feature will be implemented}_
+| `*`      | User                                                      | be able to adjust the window size                                             | I can see the data more clearly.                                                                                                      |
+| `*`      | Efficient user who is familiar with CLI tools             | refer to recent commands using up and down arrows                             | I can save time from retyping past commands                                                                                           |
+| `*`      | User with poor eyesight and strong personal preference    | edit font size and style                                                      | I can read contents on the application better                                                                                         |
+| `*`      | User who likes to stay on top of tasks                    | have a short note taking session to take down notes                           | I do not need to remember small details.                                                                                              |
+| `*`      | Admin in charge of sending emails                         | be able to export a group of emails such as overdue emails based on templates | I can craft personalized emails without the need to draft them individually                                                           |
+| `*`      | Efficient librarian                                       | continue where I left off from the previous days                              | I can continue to deliver to my borrowers the books they wanted                                                                       |
+| `*`      | Librarian in charge of placing book orders                | view statistics for different book categories                                 | I can make important decisions for the library such as getting more copies of a certain book/getting more books of a certain category |
+| `*`      | Book recommender who used the application for a long time | recommend books using stored statistics of most frequently requested books    | I can make better recommendations for students that come to us                                                                        |
+| `*`      | Impatient user                                            | be able to experience reasonable response time while using the application    | I can do my job more efficiently                                                                                                      |
+| `*`      | Easily overwhelmed request handler                        | be able to prioritize my book requests                                        | I can effectively handle requests                                                                                                     |
```

Comment:
LGTM! 

Just one comment, will we need to change the priorities of the user stories here as currently, all the user stories have low priority.

Reply:
I think eventually down the road we have to change the priority of the user stories here, but for low let's just leave as all the same priority - low priority, because I think we need to decide as a team on which stories should have a higher priority.

Reply:
Sounds good to me!